openpype/pipeline/colorspace.py

```diff
@@ -335,25 +342,60 @@ def get_imageio_config(
         anatomy_data = get_template_data_from_session()
```

Comment:
I think the docstring for this function **Returns** is wrong? It's not returning `bool`,  right? It also never returns None.